Description: Rename fixed options
 Rename system wide fixed options configuration file for consistency.
 .
 Originally appeared in version 1.3.0-1 or earlier.  (This is the earliest
 version on snapshot.debian.org, and this patch isn't mentioned in the
 changelog.)  Converted to quilt for version 1.3.0+20120605-1.
Author: Marcelo E. Magallon <mmagallo@debian.org>
Last-Update: 2014-09-12

--- a/wmppp/wmppp.c
+++ b/wmppp/wmppp.c
@@ -415,7 +415,7 @@
 	strcat(temp, "/.wmppprc");
 	parse_rcfile(temp, wmppp_keys);
 
-	strcpy(temp, "/etc/wmppp.fixed");
+	strcpy(temp, "/etc/wmppprc.fixed");
 	parse_rcfile(temp, wmppp_keys);
 
 	/* Open the display */
